x/did/client/cli/tx.go

```diff
@@ -42,48 +37,29 @@ func GetCmdCreateDID(cdc *codec.Codec) *cobra.Command {
 			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
 			cliCtx := context.NewCLIContext().WithCodec(cdc).WithAccountDecoder(cdc)
 
-			privKey, err := genPrivKey(viper.GetBool(flagInteractive))
-			if err != nil {
-				return err
-			}
-
```

Comment:
Moved the key gen logic to the `x/did/client/crypto` package, along with the `keystore.go`.